### DISCUSSION TITLE
[Question] Back process with flet

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#2234] [Q&A]
### Question

Good morning. I am currently making an application to control a process. This process is through a serial connector controller. In my application, I have an interface where I can observe the changes in my control, what elements are being activated, what values ​​the control elements have and so on. The process is cyclical, so it may be done more than once, but the process is the same each time. How can I make my application detect these changes and make the changes to it, but in the background, depending on how the data from my controller arrives. Thank you

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by Keremik0** [comment]
Hello! 
I myself recently wrote a similar application and came across this issue. I solved this by using https://flet.dev/docs/guides/python/user-controls/.
Use the last example on the page, you can include any checks in it, etc.
I apologize for my English, I use a Google translator.

**Comment by azat385** [comment]
Have a look a discussions about [how to run flet and another app in async mode](https://github.com/flet-dev/flet/discussions/2285#discussion-6013629)
my solution is https://github.com/flet-dev/flet/discussions/2285#discussioncomment-8090843
